src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -72,12 +74,14 @@
                                               token-symbol]))
         selected-account?     (rf/sub [:wallet/current-viewing-account-address])
         send-or-bridge-params (if selected-account?
-                                {:token       token-data
-                                 :stack-id    :screen/wallet.accounts
-                                 :start-flow? true}
-                                {:token-symbol token-symbol
-                                 :stack-id     :wallet-stack
-                                 :start-flow?  true})]
+                                {:token            token-data
+                                 :bridge-disabled? (:bridge-disabled? token-data)
+                                 :stack-id         :screen/wallet.accounts
+                                 :start-flow?      true}
+                                {:token-symbol     token-symbol
+                                 :bridge-disabled? (not (send-utils/bridgeable? token-symbol))
```

Comment:
@ulisesmac - should this be done in a subscription instead? wdyt? 🤔

Reply:
If I'm not mistaken, the sub `:wallet/current-viewing-account-tokens-filtered` (used in this view) depends on `token-symbol` already adds this boolean

https://github.com/status-im/status-mobile/pull/20846/files#diff-442712259612e50c55f46c1a8916e777155ff5e4964e457061af3d8388ef6f57R414

We just need to de-structure `token-data`  and use it

Quick thoughts: Why do we not hide the option to bridge instead of disabling it?

Reply:
@J-Son89 Both are fine but I'd prefer having it inside the subscription.

If it exists in the subscription, it'll be easier to be reused and the logic would be centralized.

Reply:
:wallet/current-viewing-account-tokens-filtere does, but it won't work for the home page.

I figured disabling is better as when the list is empty at least the user will see something on the page of assets to select / it doesn't look like assets are disappearing then.